hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -127,24 +131,25 @@ public void commit(TableMetadata base, TableMetadata metadata) {
       } else {
         final long currentTimeMillis = System.currentTimeMillis();
         tbl = new Table(tableName,
-                database,
-                System.getProperty("user.name"),
-                (int) currentTimeMillis / 1000,
-                (int) currentTimeMillis / 1000,
-                Integer.MAX_VALUE,
-                storageDescriptor(metadata),
-                Collections.emptyList(),
-                new HashMap<>(),
-                null,
-                null,
-                ICEBERG_TABLE_TYPE_VALUE);
+            database,
+            System.getProperty("user.name"),
+            (int) currentTimeMillis / 1000,
+            (int) currentTimeMillis / 1000,
+            Integer.MAX_VALUE,
+            storageDescriptor(metadata),
+            Collections.emptyList(),
+            new HashMap<>(),
+            null,
+            null,
+            ICEBERG_TABLE_TYPE_VALUE);
       }
 
       tbl.setSd(storageDescriptor(metadata)); // set to pickup any schema changes
       final String metadataLocation = tbl.getParameters().get(METADATA_LOCATION_PROP);
       if (!Objects.equals(currentMetadataLocation(), metadataLocation)) {
-        throw new CommitFailedException(format("metadataLocation = %s is not same as table metadataLocation %s for %s.%s",
-                currentMetadataLocation(), metadataLocation, database, tableName));
+        String errMsg = String.format("metadataLocation = %s is not same as table metadataLocation %s for %s.%s",
```

Comment:
Same, CommitFailedException supports formatting. We do not need to wrap it with `String.fromat`